pkg/apis/options/providers.go

```diff
@@ -239,6 +244,11 @@ type LoginGovOptions struct {
 	PubJWKURL string `json:"pubjwkURL,omitempty"`
 }
 
+type MastodonOptions struct {
+	// The base URL of the Mastodon instance
+	URL string `json:"url,omitempty"`
```

Comment:
This flag is required to be set, but I'm not sure where the best place is to add the validation code. Guidance would be very welcome!